feature/modules/src/main/resources/modules/system/layers/fuse/org/wildfly/extension/camel/main/module.xml

```diff
@@ -43,6 +49,8 @@
     <module name="org.slf4j" />
     <module name="ibm.jdk.ext" export="true" />
     <module name="sun.jdk.ext" export="true" />
+    <module name="javax.servlet.api" />
     <module name="org.jboss.gravia" />
+    <module name="org.wildfly.extension.undertow" />
```

Comment:
Do we need this circular `org.wildfly.extension.undertow` dependency?

Reply:
Let me check

Reply:
Why do you think it is circular?

Reply:
My bad. I got confused with `org.wildfly.extension.camel.undertow`.